src/components/Swap/Price.tsx

```diff
@@ -35,16 +39,25 @@ export default function Price({ trade, outputUSDC }: PriceProps) {
     }
   }, [base, executionPrice, inputAmount, outputAmount, outputUSDC])
 
+  return [
+    `${1} ${price.baseCurrency.symbol} = ${formatPrice(price).substring(1)} ${price.quoteCurrency.symbol}`,
+    usdcPrice && formatCurrencyAmount(usdcPrice, NumberType.FiatTokenPrice),
+  ]
```

Comment:
is there an opportunity to memoize here ? not sure if it's important, but it is calling to formatting functions which do some work that we don't want to duplicate

Reply:
oooh good point ty